### ISSUE TITLE
Crash on 2nd day. Food BK Party Consumption Model

### ISSUE THREAD

**Issue by aarondcull** [R-Vaccari/bannerlord-banner-kings#164]
On the 2nd day of playing BK (first daily tick), I get a crash about the party consumption model. 
Crash report attached.

https://report.butr.link/692F17.html

![BK Food Crash Report pic](https://github.com/R-Vaccari/bannerlord-banner-kings/assets/64914471/326d473d-1e98-4ae3-a15d-6333b1db992f)

**Describe the bug**

**Mod Version**
- Standard or CEK version
- Numeric Version: 1.2.5.x

**Evidence**
- [X] The mod is present in the callstack;
- [X] The issue only happens with the mod activated;
- [X] You tested it with the least amount of other mods as possible.

**Crash Report**
If the issue is a crash, not posting a report or callstack is wasting everyone's time.

**Comment by Sheikifang** [comment]
Did you ever find how to resolve this issue??

**Comment by aarondcull** [comment]
@Sheikifang No, sorry I didn't. I was trying to play BK with a ton of mods, that didn't work at all, and then I trimmed way down on the number of mods and this particular crash did not occur again. I never took the time to isolate which mod caused this crash.
